project.clj

```diff
@@ -14,4 +14,8 @@
                     [org.slf4j/jul-to-slf4j "1.7.7"]
                     [org.slf4j/jcl-over-slf4j "1.7.7"]
                     [org.slf4j/log4j-over-slf4j "1.7.7"]
-                    [criterium "0.4.3"]]}})
+                    [criterium "0.4.3"]]}}
+  :repositories {"farmlogs-internal"
```

Comment:
thats what I thought that it is a public repo.
but then what is this change:

```:url s3p://fl-maven-repo/mvn```

isn't this super specific to farmlogs?